packages/openactive-integration-tests/test/flows/book-and-cancel/book-and-customer-cancel-success/book-shared.js

```diff
@@ -180,7 +180,7 @@ function performTests(dataItem) {
       );
     });
 
-    sharedValidationTests.shouldBeValidResponse(() => state.ordersFeedUpdate.body.data, "Orders feed", logger, {
+    sharedValidationTests.shouldBeValidResponse(() => state.ordersFeedUpdate.body, "Orders feed", logger, {
```

Comment:
any reason why you are not using `state.ordersFeedUpdate` instead of `state.ordersFeedUpdate.body`?